pkgs/by-name/op/openclonk/package.nix

```diff
@@ -51,52 +60,47 @@ stdenv.mkDerivation {
     })
   ];
 
-  enableParallelInstalling = false;
```

Comment:
Don't remove that, compressing the groups in parallel is broken and will produce weirdly mixed files.

Reply:
Hmm, might be fine then if that serializes things with ninja.

The problem is that c4group puts its temporary files all in the same directory, so if two c4group processes run in parallel and are processing files with the same name (which is pretty common in the game files), you end up with mixed files from different groups, often without failing the build.